README.md

```diff
@@ -1,12 +1,5 @@
-# template-project
+# Give context to solver errors in Solved Python Packages
```

Comment:
Can we add more details and links to Solver, Solver dataset, some small description. Moreover for the description of te notebook some explanation about inputs and outputs in summary (with algorithms selected)

Reply:
Yes done. Please have a look and let me know what you think.